assets/src/edit-story/components/panels/alignment/alignment.js

```diff
@@ -191,30 +191,39 @@ function ElementAlignmentPanel({ selectedElements, pushUpdate }) {
     [currentButton, setCurrentButton]
   );
 
-  const backwardDirection = isRTL ? 1 : -1;
-  const forwardDirection = isRTL ? -1 : 1;
-
-  useKeyDownEffect(ref, 'left', handleNavigation(backwardDirection), [
-    handleNavigation,
-  ]);
-  useKeyDownEffect(ref, 'right', handleNavigation(forwardDirection), [
-    handleNavigation,
-  ]);
+  useKeyDownEffect(
+    ref,
+    'left',
+    () => {
+      isRTL ? handleNavigation(1) : handleNavigation(-1);
+    },
```

Comment:
This and below can also be compacted:

```suggestion
    () => handleNavigation(isRTL ? 1 : -1),
```